docs/simulation/panels/panel-export-options.md

```diff
@@ -98,7 +100,9 @@ See
 
 ## Intensity units
 
-Intensity units of exported data:
+They are the intensity units of exported data.
+
+Intensity data can be exported in:
 * Photon counts (pc)
 * Electron counts (ec)
```

Comment:
Instead of electron counts? Because I already struggle with camera parameters units tbh.. in some noise equations units don't equilibrate. I'm not sure what is the difference between electron counts and image counts now.

Reply:
Instead of electron counts? Or additionally? Because I don't really see the difference between electron counts and image counts. I already struggle with noise equations where units don't equilibrate

Reply:
Ok. Here is my input.

Depending on the particular step in the simulation, we have photons, electrons or image counts.
1. Poisson noise or shot noise of the photon emission (fluorophore level) = photon counts
2. Camera noise level without A/D conversion = electron counts
3. Camera noise level with A/D conversion = image counts

The latter two depend on the particular noise model. Two examples:
A. If we use the Gaussian noise, we basically have a poissonian distributed photon noise which gives Gaussian distributed electon counts.  Here electron counts equal image counts.
B. If we use realistic camera noise like the Hirsch et al model, we have electron counts for the detection level, the amplification level and the read-out level, the final A/D conversions gives image counts.

Reply:
Ok. Here is my input.

Depending on the particular step in the simulation, we have photons, electrons or image counts.
1. Poisson noise or shot noise of the photon emission (fluorophore level) = photon counts
2. Camera noise level without A/D conversion = electron counts
3. Camera noise level with A/D conversion = image counts

The latter two depend on the particular noise model. Two examples:
A. If we use the Gaussian noise, we basically have a poissonian distributed photon noise which gives Gaussian distributed electon counts.  Here electron counts equal image counts.
B. If we use realistic camera noise like the Hirsch et al model, we have electron counts for the detection level, the amplification level and the read-out level, the final A/D conversions gives image counts.

Reply:
Ok I understand now, thank you.
Then, how does it make sense to convert , for example, Gaussian EC to PC with a simple linear transformation? Shouldn't we export only Poisson-distributed values when PC units are chosen? Without camera contribution at all?

Reply:
Indeed, the conversion does not always make sense. According to the individual models we provide:
no camera noise: photon counts should have a conversion factor 1 or the respective detection efficiency to electron counts. Electron counts equal image counts. Thus, each photon emitted has a eta chance to be detected as electron which is shown as image counts.
Poisson noise: A Poisson from a Poisson is a Poisson. Thus, same as for no camera noise.
Gaussian noise: A Gaussian from a Poisson is Gaussian(Poisson). Thus, same as for no camera noisse:
Hirsch Model and NExpN and similar: Now we have certain transfer functions. Thus, photons are transfered into electron and finally into image counts. This is our PC to EC conversion and vice versa. The respective equations are listed in Börner et al PONE 2018 and are already well implemented, right?
Summary: The transferfunction PC to EC and vice versa is dependent on the noise model, the camera type etc. Some people are not convinced that one can re-calculate or  transfer photons from electron counts. We think, that under the assumption of a well characterized camera, one can always convert EC or IC into PC. 
EDIT: EC and IC are in many cases indifferent. Thus, a conversion is somehow artificial. However, an Andor camera shows IC and one can recalculate both EC and PC.